multisig/src/setup.rs

```diff
@@ -391,4 +412,20 @@ pub trait SetupModule:
             .set_max_tx_batch_block_duration(new_max_tx_batch_block_duration)
             .sync_call();
     }
+
+    #[only_owner]
+    #[payable("*")]
+    #[endpoint(bridgedTokensWrapperDepositLiquidity)]
```

Comment:
why this? The owner of the wrapper is not the multisig contract

Reply:
Removed.